src/components/app/content/index.js

```diff
@@ -3,18 +3,20 @@ import './index.css';
 import CardList from './cardList';
 import AddCardModal from './AddCardModal';
 import CardsPanel from './CardsPanel';
+import {useSelector} from "react-redux";
 
 function Content() {
     const [readOnly, setReadOnly] = useState(false);
 
     const [addCardModalVisible, setAddCardModalVisible] = useState(false);
     const toggleAddCardModal = () =>
         setAddCardModalVisible((prevState) => !prevState);
+    const readOnly = useSelector(state => state.cards.readOnly)
```

Comment:
You already have `readOnly` as a state. Don't you run the app locally before sending? It should have failed on your local as well.

![image](https://user-images.githubusercontent.com/52723192/113778121-b1157880-9734-11eb-961c-8c002515f6d4.png)